src/routes/stats.ts

```diff
@@ -1,7 +1,14 @@
+import { viewController } from "../controllers/api/viewController";
 import { uploadController } from "@/src/controllers/stats/uploadController";
+
 import express from "express";
 
 const statsRouter = express.Router();
 
+// get
+statsRouter.get("/view.php", viewController);
+
+// post
```

Comment:
routes are self-explanatory 

```suggestion
statsRouter.get("/view.php", viewController);
```